overrides/config-overrides/expert/betterquesting/DefaultQuests.json

```diff
@@ -55623,6 +55640,65 @@
           "taskID:8": "bq_standard:retrieval"
         }
       }
+    },
```

Comment:
Note: Port QB doesn't add this quest visually to the quest book, only adds the quest in the database. It also doesn't rearrange any quests. Please import the HM qb to perform those changes.